src/components/HotspotConfigurationPicker.tsx

```diff
@@ -0,0 +1,228 @@
+import { Alert, TextInput, TouchableWithoutFeedback } from 'react-native'
+import React, { useEffect, useRef, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import HeliumActionSheet from './HeliumActionSheet'
+import Box from './Box'
+import Text from './Text'
+import TouchableOpacityBox from './TouchableOpacityBox'
+import InfoIcon from '../assets/images/info-hollow.svg'
+import { decimalSeparator, groupSeparator, locale } from '../utils/i18n'
+import { useColors } from '../theme/themeHooks'
+
+export type AntennaId =
+  | 'helium_us'
+  | 'helium_eu'
+  | 'rak_hotspot_us'
+  | 'rak_hotspot_eu'
+  | 'nebra_outdoor'
+  | 'nebra_indoor'
+  | 'bobcat'
+  | 'syncrobit_us'
+  | 'syncrobit_eu'
+  | 'rak_custom'
+  | 'custom'
+export type Antenna = { id: AntennaId; gain: number }
+export const Antennas: Record<AntennaId, Antenna> = {
+  helium_us: { id: 'helium_us', gain: 1.2 },
+  helium_eu: { id: 'helium_eu', gain: 2.3 },
+  rak_hotspot_us: { id: 'rak_hotspot_us', gain: 2.3 },
+  rak_hotspot_eu: { id: 'rak_hotspot_eu', gain: 2.8 },
+  nebra_outdoor: { id: 'nebra_outdoor', gain: 3 },
+  nebra_indoor: { id: 'nebra_indoor', gain: 2 },
+  bobcat: { id: 'bobcat', gain: 4 },
+  syncrobit_us: { id: 'syncrobit_us', gain: 1.2 },
+  syncrobit_eu: { id: 'syncrobit_eu', gain: 2.3 },
+  rak_custom: { id: 'rak_custom', gain: 5.8 },
+  custom: { id: 'custom', gain: 1 },
+}
+
+type Props = {
+  onAntennaUpdated: (antenna: Antenna) => void
+  onGainUpdated: (gain: number) => void
+  onElevationUpdated: (elevation: number) => void
+  selectedAntenna: Antenna
+  outline?: boolean
+}
+const HotspotConfigurationPicker = ({
+  selectedAntenna,
+  onAntennaUpdated,
+  onGainUpdated,
+  onElevationUpdated,
+  outline,
+}: Props) => {
+  const { t } = useTranslation()
+  const colors = useColors()
+
+  const gainInputRef = useRef<TextInput | null>(null)
+  const elevationInputRef = useRef<TextInput | null>(null)
+
+  const [elevation, setElevation] = useState<string>()
+  const [gain, setGain] = useState<string>(
+    selectedAntenna.gain.toLocaleString(locale, {
+      maximumFractionDigits: 1,
+      minimumFractionDigits: 1,
+    }),
+  )
+
+  const onSelectAntenna = (value: string | number, index: number) => {
+    const antenna = Object.values(Antennas)[index]
+    onAntennaUpdated(antenna)
+    onGainUpdated(antenna.gain)
+    setGain(
+      antenna.gain.toLocaleString(locale, {
+        maximumFractionDigits: 1,
+        minimumFractionDigits: 1,
+      }),
+    )
+  }
+
+  const showElevationInfo = () =>
```

Comment:
Is this placeholder?

Reply:
Its the alerts that pop up when you hit the i icons near the elevation and gain text. Shows an alert with some info.